src/core/killall.c

```diff
@@ -77,13 +77,13 @@ static bool ignore_proc(pid_t pid, bool warn_rootfs) {
         return true;
 }
 
-static void wait_for_children(Set *pids, sigset_t *mask, usec_t timeout) {
+static int wait_for_children(Set *pids, sigset_t *mask, usec_t timeout) {
```

Comment:
btw, would be a cool to have a comment somewhere near the top of this function saying that this returns the number of processes remaining.